assets/queries/terraform/aws/s3_bucket_without_enabled_mfa_delete/test/negative2.tf

```diff
@@ -0,0 +1,14 @@
+resource "aws_s3_bucket" "positive6" {
```

Comment:
@rafaela-soares why are you repeating the exact same samples we already have for the resource `aws_s3_bucket` and not including any samples related to the module query update?

Reply:
My bad, I am sorry. I thought I was using a module template.